libs/ui/src/hooks/smartcard_auth/use_inserted_smartcard_auth.ts

```diff
@@ -42,20 +42,21 @@ import {
   parseUserFromCardSummary,
 } from './auth_helpers';
 import { usePrevious } from '../use_previous';
+import { areVvsg2AuthFlowsEnabled } from '../../config/features';
 
 export const VOTER_CARD_EXPIRATION_SECONDS = 60 * 60; // 1 hour
 
-// Types for the useInsertedSmartcardAuth hook
-export interface AuthScope {
+interface InsertedSmartcardAuthScope {
+  allowAdminsToAccessMachinesConfiguredForOtherElections?: boolean;
```

Comment:
I need this toggle because admins can access VxMarks configured for other elections. VxMark already handles the case that the machine is configured for another election by allowing the admin to re-program the machine with the election on their card.